src/main/java/build/buildfarm/server/BuildFarmServer.java

```diff
@@ -121,7 +120,7 @@ public synchronized void start(ServerBuilder<?> serverBuilder, String publicName
         .addService(new PublishBuildEventService())
         .intercept(TransmitStatusRuntimeExceptionInterceptor.instance())
         .intercept(headersInterceptor);
-    handleGrpcMetricIntercepts(serverBuilder);
+    GrpcMetrics.handleGrpcMetricIntercepts(serverBuilder, configs.getServer().getGrpcMetrics());
```

Comment:
We already have a BuildfarmConfigs object which contains GrpcMetrics. You can simply do configs.getGrpcMetrics.handleGrpcMetriIntercepts here and make the method non-static.